scripts/generateMetadata.js

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
```

Comment:
@manvoid Couple of questions/notes:

* Should we add this generated file to the repo? Or wouldn't it be better that it's git ignored? We would have to do an `fs.promises.readFile()` but that should be fine.
* If we do leave the file in, could we make sure that we run this script in CI and it doesn't result to any changes to the JSON file? ([similar to this](https://github.com/pinterest/gestalt/blob/master/scripts/test.sh#L35-L44))
* How does this work in development? If you add a prop, does it automatically update the JSON?
* How fast is the script?

Reply:
- I added the file to gitignore.
- In dev if a file inside gestalt/src is changed the metadata is generated again
- The script takes 2.5 second aprox